test/OrchardCore.Tests/Email/EmailTests.cs

```diff
@@ -14,7 +14,7 @@ public async Task SendEmail_WithToHeader()
             {
                 To = "info@oc.com",
                 Subject = "Test",
-                Body = "Test Message"
+                Body = new MailMessageBody { Text = "Test Message" }
```

Comment:
How about an implicit cast operator in `MailMessageBody` from `string` (fills `Text`) and `IHtmlContent` (fills `Html`)? Then here you could write `Body = "Test Message"`.